app/components/hydrogen/Text.jsx

```diff
@@ -0,0 +1,83 @@
+import clsx from 'clsx';
+import {missingClass, formatText} from '~/lib/utils';
+
+export function Text({
+  as: Component = 'span',
+  className,
+  color = 'default',
+  format = false,
+  size = 'copy',
+  width = 'default',
+  children,
+  ...props
+}) {
+  const colors = {
+    default: 'inherit',
+    primary: 'text-primary/90',
+    subtle: 'text-primary/50',
+    accent: 'text-accent',
+    contrast: 'text-contrast/90',
+  };
+
+  const sizes = {
+    lead: 'text-lead font-medium',
+    copy: 'text-copy',
+    fine: 'text-fine subpixel-antialiased',
+  };
+
+  const widths = {
+    default: 'max-w-prose',
+    narrow: 'max-w-prose-narrow',
+    wide: 'max-w-prose-wide',
+  };
+
+  const styles = clsx(
+    missingClass(className, 'max-w-') && widths[width],
+    missingClass(className, 'whitespace-') && 'whitespace-pre-wrap',
```

Comment:
```suggestion
```
I'm going to remove this class. The text content has the whitespace stripped as part of a build step which minifies the code, making this class useless for this component, but we don't apply any minification in the running editor (for the sake of debugging etc.), which means that this class actually causes alignment discrepancies when running in the editor